content/en/events/2024/kcsna/faq.md

```diff
@@ -0,0 +1,117 @@
+
+We welcome everyone to start contributing to the project, but with 1600+
+Kubernetes Org members and limited space and budget requirements we've
+made the decision to limit attendance to those that are already actively
+engaged with the project.
+
+If you are not yet a member, but active within the project, please consider
+joining! The process of joining is straight forward. See our [org membership]
+docs for more information. If your application to join is pending, please
+contact us at summit-team@kubernetes.io about registering.
+
+Kubernetes Orgs:
+
+<ul>
+<li><a href="https://github.com/kubernetes" rel="noopener noreferrer" target="_blank">kubernetes</a></li>
+<li><a href="https://github.com/kubernetes-client" rel="noopener noreferrer" target="_blank">kubernetes-client</a></li>
+<li><a href="https://github.com/kubernetes-csi rel="noopener noreferrer" target="_blank">kubernetes-csi</a></li>
+<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
+</ul>
+
+
+[org membership]: https://github.com/kubernetes/community/blob/master/community-membership.md#member
+
+### What is a sponsored attendee?
+
+If you are a SIG lead or subproject lead and there is someone that is not yet
+an org member or from another project that might take part in a panel,
+SIG face-to-face meeting etc, you can sponsor them to attend the summit.
+Please [email us] to make arrangements.
+
+### I have an idea for a session
+
+Most of the Summit will be unconference-style, so you will have an opportunity
+to propose a session closer to the Summit itself. If you have an idea for a talk,
+workshop or panel please post your idea to the unconference issue or the CFP
+form once they become available.  
+
+### My SIG wants to meet at the Summit
+We will have some space for SIG/WG meetings during the summit, please submit
+your request via the CFP form once it becomes available.
+
+### Will there be a doc sprint?
+
+Yes, SIG-Docs plans to hold a Doc Sprint at the Summit, which will run for the
+full day from 11am to 4pm.
+
+### Will there be a New Contributor Workshop?
+
+No, they are very time, person and resource intensive. We encourage those
+interested in becoming a contributor to use the New Contributor Course 
+slated to launch later this year.
+
+Until then, here are some additional resources you may find useful:
+- [New Contributor Workshop 2019 - Barcelona](https://www.youtube.com/watch?v=BQ7y2TFOzF4&list=PL69nYSiGNLP2WTJ6P8sQenhf0RY-JqF5L)
+- [New Contributor Workshop 2019 - San Diego](https://www.youtube.com/watch?v=uUJrGwAom-E&list=PL69nYSiGNLP0OWp38tPBc-jSlMmwWr6Ci&index=15)
+- [Kubernetes Contributor Guide](/docs/guide/)
+
+### Can I bring a guest to the Social?
+
+Contributors will be allowed to bring a single family member
+or partner to the social (a "plus one"). 
+[See the Social page to request a guest pass](https://www.kubernetes.dev/events/2024/kcsna/social/).
+
```

Comment:
```suggestion

### What about Utah HB 257?

Recent laws passed by the state of Utah may cause problems for some members of our community.  Please see the [legal information] on the Venue page.

[legal information]: link-to-Laws-section-of-venue-page
```

Reply:
I believe we should link to https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/attend/diversity-inclusion/#utah-bathroom-bill-statement, but I think making a statement like "may cause problems for some members of our community" should probably be kept out of it. That statement somewhat conflicts with the text on the referenced page:

> To clarify the legislation, this law does not extend to privately owned venues, including those hosting our conference and related activities

Better to make sure folks have easy access to the vetted information than to make any statements here.